### PR TITLE
add python japronto

### PR DESCRIPTION
**Pull request by sharkspeed**
Add a new Python framework: [japronto](https://github.com/squeaky-pl/japronto)

### CODE REVIEW

**Comment by tbrand** [comment]
Could you add japronto to the Makefile or neph.yml?

**Comment by sharkspeed** [comment]
Oh, I forgot to add Makefile, my fault.

File: neph.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-main:
+ggmain:
```
Comment (tbrand) [inline neph.yml]: Oops, it seems that you forgot press Ctrl-key on your emacs? 😄

Comment (sharkspeed) [inline neph.yml]: ah, I'm not familar with neph.yml. How can i fix that? I don't think i have modified this file.

Comment (tbrand) [inline neph.yml]: By MYSELF :rofl: Sorry...
Please merge the latest.:pray:

**Comment by tbrand** [comment]
Excellent commit again!
I've confirmed that it works!!:tada:

**Comment by tbrand** [comment]
This commit has been included new release:tada:( https://github.com/tbrand/which_is_the_fastest/releases/tag/v1.1.2 )
Thanks again and waiting for your new commit! :smile:

**Pull request merged**